src/Scene/NodeProcess.js

```diff
@@ -95,36 +95,43 @@ NodeProcess.prototype.subdivideNode = function(node, camera, params) {
                 var j;
 
                 child.matrixSet = [];
-
+                let tileMatrixSet,texturesCount;
```

Comment:
Nope. Replacing `var` by `let` is a good idea but don't move the declaration to a bigger scope.